app/models/payment_state.rb

```diff
@@ -40,25 +40,30 @@ def ttl_paid
   end
 
   def total_cost
-    cost_sessions_ttl + cost_activities_ttl + CampConfiguration.active_camp_fee_cents
+    base_cost = cost_sessions_ttl + cost_activities_ttl
+    active_camp = CampConfiguration.active.first
+    if active_camp&.application_fee_required
```

Comment:
total_cost should be counted differently for applicants who applied when application_fee_required was true and for those who applied after application_fee_required was changed to false.
Right now if I make application_fee_required = false balance_due for the old applicants is displayed as $-100. Please check the Active Admin enrollments pages.

Reply:
This is handled with line 45 statement
    if active_camp&.application_fee_required

Reply:
What is handled? Please see the screenshot.
I turned the application_fee_required off after four applications were submitted and registered another applicant.
See the Balance Due
<img width="1782" alt="Screenshot 2024-12-12 at 2 19 57 PM" src="https://github.com/user-attachments/assets/7b55a718-df6e-42eb-a024-47a4b8f1cd19" />

Reply:
I just pushed a set of updates to capture the items missed.